Ui update

Philip-Scott opened the pull request:
Fixes #3

Philip-Scott committed:
UI update

Philip-Scott committed:
UI update

Philip-Scott committed:
Update Indicator.vala

MarcusWichelmann commented on src/Indicator.vala:
```diff
@@ -52,23 +52,28 @@ public class Drop.Indicator : Wingpanel.Indicator {
+			    try {
+			         show_notification (_("Incoming file from: %s").printf(transmission.get_client_name ()));
+			    } catch (Error e) {
+			        stderr.printf ("Transmission error: %s", e.message);
```
Better uses warning(). Also use a better error description.

MarcusWichelmann commented on src/Indicator.vala:
```diff
@@ -52,23 +52,28 @@ public class Drop.Indicator : Wingpanel.Indicator {
+
 			out_list.transmission_added.connect ((transmission) => {
-				chech_visibility ();
+				check_visibility ();
```
:joy:

MarcusWichelmann commented on src/Indicator.vala:
```diff
@@ -82,34 +87,40 @@ public class Drop.Indicator : Wingpanel.Indicator {
+			this.notification.update ("Drop", message, NOTIFICATION_ICON);
 			try {
 				this.notification.show ();
 			} catch (Error E) {}
```
the E needs to be lower case.
Add a warning with descriptive text.

MarcusWichelmann commented on src/Indicator.vala:
```diff
@@ -82,34 +87,40 @@ public class Drop.Indicator : Wingpanel.Indicator {
-		else if (!visible) visible = true;
+	private void check_visibility () {
+		if (in_list.transmission_count == 0 && out_list.transmission_count == 0) {
+		    close ();
```
Please use this.close() and this.visible etc. to make more clear where the one who reads this code can find the method.

Philip-Scott committed:
Update Indicator.vala

MarcusWichelmann commented on src/Indicator.vala:
```diff
@@ -52,23 +52,28 @@ public class Drop.Indicator : Wingpanel.Indicator {
-				show_notification ("New Incoming File");
-				chech_visibility ();
+				try {
+					show_notification (_("Incoming file from: %s").printf(transmission.get_client_name ()));
```
space after printf

Philip-Scott committed:
Spaaaace

![spaaace](http://img03.deviantart.net/5904/i/2014/004/6/5/spaceeee__by_baka_chanlove-d70ueo7.png)

MarcusWichelmann commented:
Looks good. Ready for merging when the other branch is ready, too.
Thanks for your work! :smile:
